src/components/charts/RangeBar.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropType from "prop-types";
+import {
+  ComposedChart,
+  Bar,
+  XAxis,
+  YAxis,
+  Scatter,
+  ReferenceArea,
+  Label,
+} from "recharts";
+
+const RangeBar = ({ width = 150, height = 24, value, min = 0, max = 1.2}) => {
+  const data = [{
+    name: "value1",
+    max: max,
+    value: value,
+  }];
+
+  return (
+    <ComposedChart
+      width={width}
+      height={height}
+      data={data}
+      layout="vertical"
+      margin={{
+        top: 4,
+        left: 0,
+        bottom: 0,
+        right: 0,
+      }}
+    >
+      <XAxis type="number" domain={[min, max]} hide={true} />
+      <YAxis type="category" dataKey="name" hide={true} />
+      <Bar dataKey="value"
+        barSize={height - 6}
+        shape={renderGradientShape}
+        isAnimationActive={false}
+      />
+      <Scatter dataKey="value"
+        shape={renderDashedLine}
+        isAnimationActive={false}
+      />
+      <Scatter dataKey="value"
+        shape={renderTriangleShape}
+        isAnimationActive={false}
+      />
+      <ReferenceArea x1="0" x2={data.max} fill="transparent">
+        <Label value="LOW" position="insideLeft" fontSize={10} fontWeight={500} />
```

Comment:
The "LOW" and "HIGH" strings should come from propsMapping, just like `min` and `max`. Same with the gradient's stop colors. Those may vary from metric to metric.

That said, it'd be good to have "LOW" and "HIGH" and the current colors as defaults in case those aren't defined in propsMapping.js.